src/components/Create/components/pages/hooks/FormDispatchWatch.ts

```diff
@@ -1,16 +1,17 @@
 import { ActionCreatorWithPayload } from '@reduxjs/toolkit'
 import { FormInstance } from 'antd'
 import { useWatch } from 'antd/lib/form/Form'
-import { useAppDispatch } from 'redux/hooks/AppDispatch'
 import isEqual from 'lodash/isEqual'
 import { useEffect } from 'react'
+import { useAppDispatch } from 'redux/hooks/AppDispatch'
 
 /**
  * Watches a form field and updates a redux state when it changes.
  */
 export const useFormDispatchWatch = <
```

Comment:
just a little typing update on this to make it play nicely with a FormFields with field values of multiple types